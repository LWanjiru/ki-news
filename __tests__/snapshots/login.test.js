import React from 'react';
// import { shallow } from 'enzyme';
import to_json from 'enzyme-to-json';
import Login from '../../src/scripts/components/login';
import renderer from 'react-test-renderer';

describe('Login component snapshots', () => {
  it('renders a snapshot', () => {
    const body = renderer.create(<Login/>).toJSON();
    expect(body).toMatchSnapshot();
  });
});
