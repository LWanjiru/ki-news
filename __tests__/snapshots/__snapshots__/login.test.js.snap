// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component snapshots renders a snapshot 1`] = `
<div>
  <div>
    <img
      src="../../public/images/logo.jpg"
    />
    &gt;
  </div>
  <div>
    <h1>
      KI-All News
    </h1>
  </div>
  <div>
    <p>
      Welcome to the Know It All news website, and be informed on what is happening around the world. To continue to the News page, please Login below.
    </p>
  </div>
  <div>
    <button>
      Signin with Google+
    </button>
  </div>
</div>
`;
